docs/guides/code_examples/storage_clients/sql_storage_client_basic_example.py

```diff
@@ -0,0 +1,10 @@
+from crawlee.crawlers import ParselCrawler
+from crawlee.storage_clients import SQLStorageClient
+
+
+async def main() -> None:
+    # Create a new instance of storage client.
+    # Use the context manager to ensure that connections are properly cleaned up.
+    async with SQLStorageClient() as storage_client:
```

Comment:
Please also mention that this will use sqlite and create a .db file for you (IIRC)